.gitattributes

```diff
@@ -0,0 +1 @@
+unidist/_version.py export-subst
```

Comment:
Should we also add `* text=auto` as Modin does?

Reply:
Yes, we should.

Reply:
added